.github/workflows/release.yaml

```
@@ -0,0 +1,51 @@
+            repo_name=$(echo $cmd | awk '{print $4}')
+            if ! helm repo list | grep -q $repo_name; then
+              $cmd
+            else
+              echo "Repository $repo_name already exists"
+            fi
+          done
+          done
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.6.0
+        with:
+            skip_existing: true
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)